src/openklant/conf/base.py

```diff
@@ -76,3 +81,6 @@
     ),
     group="Celery",
 )  # soft
+
+# Notifications
+NOTIFICATIONS_DISABLED = True
```

Comment:
This is now no longer configurable via environment variables. I think you'll have to override it like this to make sure the default is different and the helptext is correct:

```suggestion
NOTIFICATIONS_DISABLED = config(
    "NOTIFICATIONS_DISABLED",
    default=True,
    help_text=(
        "indicates whether or not notifications should be sent to the Notificaties API "
        "for operations on the API endpoints."
    ),
)
```